bin/ch/WScriptJsrt.cpp

```diff
@@ -35,6 +35,7 @@
 #endif // FreeBSD or unix ?
 #endif // _WIN32 ?
 
+#pragma prefast(disable:26444, "This warning unfortunately raises false positives when auto is used for declaring the type of an iterator in a loop.")
```

Comment:
>26444 [](start = 24, length = 5)

suppress by name macro instead of number?

Reply:
No macro for this one, unfortunately.